YoCode/CodeCoverageCheck.cs

```diff
@@ -50,7 +46,8 @@ public CodeCoverageCheck(string dotCoverDir, string workingDir, FeatureRunner fe
             }
             else
             {
-                CodeCoverageEvidence.FeatureImplemented = (coverage > passPerc) ? true : false;
+                CodeCoverageEvidence.FeatureRating = ( (double) GetCodeCoverage(report) ) / 100;
+                CodeCoverageEvidence.FeatureImplemented = coverage > passPerc;
```

Comment:
Does the pass % need to be changed? Is 30% still a good value for pass/fail based on the tests we have?